rpcs3/rpcs3qt/rsx_debugger.cpp

```diff
@@ -299,6 +372,35 @@ void Buffer::ShowWindowed() const
 	//}
 }
 
+void Buffer::ShowContextMenu(const QPoint& pos)
+{
+	if (m_image.isNull())
+		return;
+
+	QMenu contextMenu(tr("Save Image"), this);
+
+	QAction action("Save Image At", this);
+	connect(&action, &QAction::triggered, this, [&]()
+	{
+		if (m_image.isNull())
+			return;
+
+		const QString path = QFileDialog::getSaveFileName(0, "Save Image", "", "Image (*.png)");
```

Comment:
needs early out if path.empty(). The user can cancel the path selection

Reply:
Fixed.